src/main/java/games/strategy/engine/framework/startup/ui/EnginePreferences.java

```diff
@@ -1,246 +0,0 @@
-package games.strategy.engine.framework.startup.ui;
```

Comment:
This class supported the engine preferences window that opened from the main screen.
The sound control was dropped for now, there is one available in game. Adding the same options to the settings window I think will make sense.

The look and feel option was moved to settings.

The set memory window button was broken!

IN the end all components were removed, we were left with launch map makers and console. I moved these to the main window and put them behind the beta feature flag. Both map makers need work, and for now it is an okay place to hide the console. Could use some suggestions for different or longer term treatments. I suspect this will be okay medium term while we evaluate the map makers and decide on any potential new home for the launch console button.